gnomad/utils/sparse_mt.py

```diff
@@ -401,6 +409,9 @@ def get_site_info_expr(
     # This is done outside of _get_info_agg_expr as the behavior is different in site vs allele-specific versions
     agg_expr["FS"] = fs_from_sb(agg_expr["SB"])
 
+    # Add SOR if SB is present
```

Comment:
Should there be a check here and also in line 408? Something similar to the check for DP in L395. The comment makes it seem like SB may not be in the entry or gvcf info fields but the module will fail in 413 without an SB field. Or maybe the comment needs to be updated to remove the "if SB..."